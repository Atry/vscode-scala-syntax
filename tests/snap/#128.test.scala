p133
p133.
p133.42
p4e1
p.2
x124x
