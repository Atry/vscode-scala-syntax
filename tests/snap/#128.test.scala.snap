>p133
#^^^^^ source.scala
>p133.
#^^^^^^ source.scala
>p133.42
#^^^^ source.scala
#    ^^^ source.scala constant.numeric.scala
>p4e1
#^^^^^ source.scala
>p.2
#^ source.scala
# ^^ source.scala constant.numeric.scala
>x124x
#^^^^^^ source.scala
>